tasks/runtime.yml

```diff
@@ -103,6 +103,20 @@
   register: ohpc_cgroup_conf
   # NB uses restart rather than reload as this is needed in some cases
 
+- name: Template mpi.conf
```

Comment:
I think this should also be added to the slurmd trigger logic [here](https://github.com/stackhpc/ansible-role-openhpc/blob/b2367a7864be5837fa2791a6694f3506b7fab90a/tasks/runtime.yml#L144C1-L144C42):

```
- name: Notify handler for slurmd restart
```

(I'd love to actually fix this :cry: )

Reply:
Well, actually, please check the slurm docs for whether this is the case. And if not, add a note.